dev/com.ibm.ws.jca.cm/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -29,7 +29,7 @@ agedTimeout=Aged timeout
 agedTimeout.desc=Amount of time before a connection can be discarded by pool maintenance. A value of -1 disables this timeout. A value of 0 discards every connection, which disables connection pooling.
 
 connTimeout=Connection timeout
-connTimeout.desc=Amount of time after which a connection request times out. A value of -1 disables this timeout.
+connTimeout.desc=Amount of time after which a connection request times out. A value of -1 disables this timeout meaning infinite wait.  A value of 0 is immediate meaning no wait.
```

Comment:
Suggested change: "Amount of time after which a connection request times out. A value of -1 disables this timeout, meaning infinite wait. A value of 0 is immediate, meaning no wait."

Add a comma before "meaning" in both places and remove the extra space before the last sentence.

Reply:
Updated